docs/src/main/tut/monoid.md

```diff
@@ -60,23 +60,9 @@ l.foldMap(i => i.toString)
 ```
 
 To use this
-with a function that produces a tuple, we can define a `Monoid` for a tuple 
+with a function that produces a tuple, cats also provides a `Monoid` for a tuple
```

Comment:
Is this potentially confusing since these actually come from Algebra and the following sentence goes into detail about the role Algebra plays here?